jekyll/_includes/snippets/runner/install-with-cli-steps.adoc

```diff
@@ -0,0 +1,32 @@
+//== Command line installation: pre-steps
+
+// Display the following step for machine runner installation only as container runner is not yet avaiable on server
+ifdef::machine[]
+NOTE: If you are installing **self-hosted runners for server**, the CircleCI CLI needs to be configured using your server API key. Run `circleci setup` to configure the CLI and access the option to supply a new API token if required.
+endif::[]
+
+In order to install self-hosted runners, you will need to create a namespace and authentication token by performing the steps listed below. Please note that to create resource classes and tokens you need to be an organization administrator in the VCS provider.
+
+You can view your installed runners in the https://app.circleci.com/[web app], by clicking *Self-Hosted Runners* on the left navigation.
+
+. Create a namespace for your organization's self-hosted runners. Please note, CircleCI will autogenerate a namespace based on your organization's name. Verify this is the namespace you want to use. *Each organization can only create a single namespace*. We suggest using a lowercase representation of your CircleCI organization's account name. If you already use orbs, this namespace should be the same namespace orbs use.
```

Comment:
If the command below is to be used, I don't get when CCI autogenerates a namespace 🤔